pkg/master/controller.go

```diff
@@ -168,7 +174,8 @@ func (c *Controller) Stop() {
 	go func() {
 		defer close(finishedReconciling)
 		klog.Infof("Shutting down kubernetes service endpoint reconciler")
-		if err := c.EndpointReconciler.StopReconciling(kubernetesServiceName, c.PublicIP, endpointPorts); err != nil {
+		defer c.EndpointReconciler.StopReconciling()
```

Comment:
dont' you want to stop reconciling before you remove the endpoints so you don't accidentally put it back?

Reply:
done